system/src/system_network_diagnostics.cpp

```diff
@@ -302,7 +302,14 @@ class NetworkCellularCellGlobalIdentityMobileNetworkCodeDiagnosticData
     {
         CellularGlobalIdentity cgi;
```

Comment:
I've just noticed this. Why aren't we zeroing out this struct and not setting neither size nor version?

Reply:
Great catch! Thank you!